scripts/CMakeLists.txt

```diff
@@ -98,3 +98,16 @@ install(
     COMPONENT
         meta
 )
+
+install(
+    FILES
+        nebula-storaged-listener.service
```

Comment:
will it have nebula-metad-listener.service?

Reply:
The enterprise version has it, but the community version does not have a meta listener